packages/tools/testTools/src/visualizationUtils.ts

```diff
@@ -232,9 +232,14 @@ export const evaluateRenderSceneForVisualization = async (renderCount: number) =
             if (window.scene.activeCamera && (window.scene.activeCamera as any).useAutoRotationBehavior) {
                 (window.scene.activeCamera as any).useAutoRotationBehavior = false;
             }
+            const sceneAdts: any[] = window.scene!.textures.filter((t: any) => t.getClassName() === "AdvancedDynamicTexture");
+            const adtsAreReady = () => {

```

Comment:
should not gui is ready be part of scene ready ?

it is strange to check this here